docs/source/executing-operations/demand-control.mdx

```diff
@@ -0,0 +1,554 @@
+# TYPE cost_delta histogram
+cost_delta_bucket{otel_scope_name="apollo/router",le="0"} 0
+cost_delta_bucket{otel_scope_name="apollo/router",le="10"} 2
+cost_delta_bucket{otel_scope_name="apollo/router",le="100"} 9
+cost_delta_bucket{otel_scope_name="apollo/router",le="1000"} 7
+cost_delta_bucket{otel_scope_name="apollo/router",le="10000"} 19
+cost_delta_bucket{otel_scope_name="apollo/router",le="100000"} 20
+cost_delta_bucket{otel_scope_name="apollo/router",le="1000000"} 20
+cost_delta_bucket{otel_scope_name="apollo/router",le="+Inf"} 20
+cost_delta_sum{otel_scope_name="apollo/router"} 21934
+cost_delta_count{otel_scope_name="apollo/router"} 1
+# TYPE cost_estimated histogram
+cost_estimated_bucket{cost_result="COST_OK",otel_scope_name="apollo/router",le="0"} 0
+cost_estimated_bucket{cost_result="COST_OK",otel_scope_name="apollo/router",le="10"} 5
+cost_estimated_bucket{cost_result="COST_OK",otel_scope_name="apollo/router",le="100"} 5
+cost_estimated_bucket{cost_result="COST_OK",otel_scope_name="apollo/router",le="1000"} 9
+cost_estimated_bucket{cost_result="COST_OK",otel_scope_name="apollo/router",le="10000"} 11
+cost_estimated_bucket{cost_result="COST_OK",otel_scope_name="apollo/router",le="100000"} 20
+cost_estimated_bucket{cost_result="COST_OK",otel_scope_name="apollo/router",le="1000000"} 20
+cost_estimated_bucket{cost_result="COST_OK",otel_scope_name="apollo/router",le="+Inf"} 20
+cost_estimated_sum{cost_result="COST_OK",otel_scope_name="apollo/router"}  
+cost_estimated_count{cost_result="COST_OK",otel_scope_name="apollo/router"} 20
+```
+
+An example chart of a histogram:
+
+<img class="screenshot" src="../images/demand-control-example-histogram.png" width="600" />
+
+
+You can also chart the percentage of operations that would be allowed or rejected with the current configuration:
+
+<img class="screenshot" src="../images/demand-control-example-cost-result.png" width="600" />
+
+Although estimated costs won't necessarily match actual costs, you can use the metrics to ascertain the following: 
+- Whether any operations have underestimated costs
+- What to set `static_estimated.list_size` as the actual maximum list size
+- What to set `static_estimated.max` as the maximum cost of an allowed operation
+
+In this example, just under half of the requests would be rejected with the current configuration. The cost of queries are also underestimated because `cost.delta` is non-zero.
+
+3. To figure out what operations are being rejected, define a telemetry custom instrument that reports when an operation has been rejected because its cost exceeded the configured cost limit:
+
+```yaml title="Example custom instrument to show rejected operations"
+telemetry:
+  instrumentation:
+    instruments:
+      supergraph:
+        # custom instrument
+        cost.rejected.operations:
+          type: histogram
+          value:
+            cost: estimated                                          # Estimated cost is used to populate the histogram
```

Comment:
With these comments on the same line, readers have to horizontal scroll the code block. Can they be above the line as in the other examples?
<img width="787" alt="image" src="https://github.com/apollographql/router/assets/23219998/1a47e442-3f5b-4c03-941c-f3244f6dcb72">